src/lib/types.ts

```diff
@@ -50,7 +50,7 @@ export type WitOracleQueryParams = {
      */
     resultMaxSize?: number,
     /**
-     * Mininum reward in $nanoWIT for very validator that positively contributes to get the Wit/Oracle
+     * Mininum reward in $pedros for very validator that positively contributes to get the Wit/Oracle
```

Comment:
An external user reading this, doesn't understand what Pedro's are
```suggestion
     * Mininum reward in $nanoWIT for very validator that positively contributes to get the Wit/Oracle
```